vmdb/app/models/configuration_profile.rb

```diff
@@ -42,4 +45,12 @@ def tag_hash
   end
 
   alias_method :manager, :configuration_manager
+
+  def total_configured_systems
+    ConfiguredSystem.count(:conditions => {:configuration_profile_id  => id})
+  end
+
+  def v_zone_name
```

Comment:
I like the way `configured_system` does it

```ruby
delegate :my_zone, :provider, :zone, :to => :manager
end
```

If it looks like zone is not properly populated from the provider, we can look into addressing that elsewhere.